src/components/ProductMilestonePages/ProductMilestonePages.tsx

```diff
@@ -175,13 +173,7 @@ export const ProductMilestonePages = ({ children }: PropsWithChildren<IProductMi
           {serviceContainerArtifacts.data?.totalHits}
         </PageTabsLabel>
       </PageTabsItem>
-      <ExperimentalContent>
-        <PageTabsItem url="interconnection-graph">
```

Comment:
Personally, until you are able to test it thoroughly locally, I would highly recommend keeping this as experimental, using `dataSource="experimental"` to align expectations with our users.